src/data/mv_polynomial/equiv.lean

```diff
@@ -144,6 +144,24 @@ def ring_equiv_congr [comm_semiring S₂] (e : R ≃+* S₂) :
   map_mul'  := ring_hom.map_mul _,
   map_add'  := ring_hom.map_add _ }
 
+/-- If `A` and `B` are isomorphic `R`-algebras, then we get an isomorphism
+`mv_polynomial S₁ A ≃ₐ[R] mv_polynomial S₁ B`. -/
+def alg_equiv_congr {A B : Type*} [comm_semiring A] [comm_semiring B] [algebra R A] [algebra R B]
+  (e : A ≃ₐ[R] B) :
+  algebra.comap R A (mv_polynomial S₁ A) ≃ₐ[R] algebra.comap R B (mv_polynomial S₁ B) :=
```

Comment:
The docs for comap seem to suggest it is deprecated in favor of is_scalar_tower

Reply:
As far I understand the docs `comap` is useful when the two algebra structures are already there, that is this case: `A` is supposed to be an `R` algebra and `mv_polynomial S₁ A` is naturally an `A`-algebra. It seems to me a little unnatural to introduce a new `R`-algebra structure on `mv_polynomial S₁ A` and immediately suppose that it is the one given by `A`.